docs/Configuration.md

```diff
@@ -973,7 +973,7 @@ The glob patterns Jest uses to detect test files. By default it looks for `.js`,
 
 See the [micromatch](https://github.com/jonschlinkert/micromatch) package for details of the patterns you can specify.
 
-See also [`testRegex` [string | array\<string>]](#testregex-string--arraystring), but note that you cannot specify both options.
+See also [`testRegex` [string | array\<string> | array\<RegExp>]](#testregex-string--arraystring), but note that you cannot specify both options.
```

Comment:
the docs are correct - you cannot _pass_ as `RegExp` here, it has to be a string.

however, internally it should be typed as you suggest, as that's what we have in memory

Reply:
Hmm I thought [this](https://github.com/ahnpnl/ts-jest-babel-example/blob/0aa3599fa6899b2f72fb8aee19471d737233d924/jest.config.js#L28) is considered as regexp object ?

Reply:
yup :+1:

Reply:
It has to be a string as we serialize the config and pass it over IPC, and there's no `toJSON` on `RegExp`. `telejson` supports this, but I haven't gotten to using it yet.

https://github.com/facebook/jest/pull/7251

Reply:
done reverting changes in `Configuration.md`